src/autopas/containers/linkedCells/LinkedCells.h

```diff
@@ -499,6 +509,8 @@ class LinkedCells : public CellBasedParticleContainer<FullParticleCell<Particle>
           (iteratorBehavior & IteratorBehavior::halo and _cellBlock.cellCanContainHaloParticles(cellIndex));
       if constexpr (regionIter) {
         // short circuit if already false
+        // TODO: should we remove the if isRelevant here, since a cell with a different ownership-state could contain a
+        // moved particle with another ownership state?
```

Comment:
```suggestion
```

No, the cells will always only contain halo or owned particles for linked cells. It might only be the case that an owned particle is in a position that is inside a halo cell.